source/includes/openstack/_instances.md

```diff
@@ -135,6 +135,10 @@ Required | &nbsp;
 `networkId`<br/>*UUID* | The network in which the instance will be created. If you don't have a network, it can be created through the create network API.
 `securityGroupNames`<br/>*Array[string]* | The list of [security groups](#security-groups) to be associated to the instance
 
+Optional | &nbsp;
+------ | -----------
+`userData`<br/>*string* | Data that the user can specify when they launch an instance. User data holds a file that cloud-aware applications in the guest instance can access. One application that uses user data is the cloud-init system, which is available on various Linux systems.
```

Comment:
I looked at https://docs.openstack.org/user-guide/cli-provide-user-data-to-instances.html to write it, it uses some of their terminology. Should I cite them or change it in some way? What is your recommendation